cdragontoolbox/binfile.py

```diff
@@ -131,6 +132,21 @@ def getv(self, key, default=None):
         except KeyError:
             return default
 
+    def hasr(self, key):
```

Comment:
What does `hasr` means? (same for `getr` and `getrv`)

Reply:
hasRaw, getRaw, and getRawValue.
I tried using get and getv first, but since they hash the string, it didnt work